### DISCUSSION TITLE
Sync buffers & lazygit

### DISCUSSION THREAD

**Discussion by aap01** [LunarVim/LunarVim#4190] [Q&A] [answered]
How to implement `automatically refresh buffers` after changes are made via `lazygit`?

**Comment by LostInTheLogs** [comment, marked as answer]
[create your own toggleterm teminal](https://github.com/LunarVim/LunarVim/blob/e548d6bc992c13e6208e5099e9773fb8245d7225/lua/lvim/core/terminal.lua#L149-L167) with custom on_close that would call `:checktime` or do something different (I'm assuming you use the lazygit mapping to use lazygit)
